deltachat-ffi/deltachat.h

```diff
@@ -2960,7 +2870,6 @@ char*            dc_chat_get_info_json       (dc_context_t* context, size_t chat
  */
 
 
-#define         DC_CHAT_ID_DEADDROP          1 // virtual chat showing all messages belonging to chats flagged with chats.blocked=2
```

Comment:
i suggest to leave the removed constants for  a while and define the removed functions as `#define dc_func(a, b, c) (1)` or so.

in theory, that should not be needed, however, in practice, it turns out several times in the past that that is useful, we even re-added already removed things, iirc for node/desktop.

Reply:
> we even re-added already removed things, iirc for node/desktop.

Last time we did it was for mailing list support release, to allow running desktop with the latest desktop core without modifications. It sort of worked because no critical functions were affected, the only problem was that for mailing lists you did not see overridden sender names.

Here you will get contact requests that cannot be used because messages cannot be sent to blocked chats and there is no `dc_unblock_chat` button in existing clients. This is why I wanted to use this as an opportunity to remove all the deprecated calls.

Reply:
k, i agree. if it turns out, that stubs are advantageous, we can re-add them as needed.